Mathlib/Analysis/SpecialFunctions/Complex/CircleAddChar.lean

```diff
@@ -126,3 +126,27 @@ noncomputable def rootsOfUnityCircleEquiv : rootsOfUnity n Circle ≃* rootsOfUn
   right_inv _ := by aesop
 
 instance : HasEnoughRootsOfUnity Circle n := (rootsOfUnityCircleEquiv n).symm.hasEnoughRootsOfUnity
+
```

Comment:
I suggest adding the following lemma:
```suggestion
@[simp] lemma rootsOfUnityCircleEquiv_apply (w : rootsOfUnity n Circle) : 
    ((rootsOfUnityCircleEquiv n w).val : ℂ) = ((w.val : Circle) : ℂ) :=
  rfl

```
This shortens the proofs below. Generally, whenever you're using `rfl` as a step in a longer proof, it's a sign of a missing simp lemma.